tensorboard/plugins/debugger/tf_debugger_dashboard/tf-op-selector.html

```diff
@@ -218,10 +236,17 @@
       }
     },
 
+    _renderHierarchyWithTimeout(watchHierarchy, debugWatchChange, filterMode, filterInput) {
+      this._clearSelectorHierarchy();
+      this.set('_isLoading', true);
+      setTimeout(() => {
```

Comment:
I believe

```typescript
this.async(() => {
  this._renderHierarchy(watchHierarchy, debugWatchChange, filterMode, filterInput);
});
```

works here.

Reply:
As per offline discussion, setTimeout is correct here. The purpose is to let the spinner DOM update by pausing for a few milliseconds fist.